client/app/components/EditInPlace.jsx

```diff
@@ -1,23 +1,26 @@
 import React from "react";
 import PropTypes from "prop-types";
+import cx from "classnames";
 import { react2angular } from "react2angular";
 import { trim } from "lodash";
+import Input from "antd/lib/input";
 
 export class EditInPlace extends React.Component {
   static propTypes = {
     ignoreBlanks: PropTypes.bool,
     isEditable: PropTypes.bool,
-    editor: PropTypes.string.isRequired,
     placeholder: PropTypes.string,
     value: PropTypes.string,
     onDone: PropTypes.func.isRequired,
+    textArea: PropTypes.bool,
```

Comment:
How about calling it `multiline`? Also, let's add `editorProps: PropTypes.object` and pass it to editor component when in editing mode - it will allow to use features like `Textarea.autoSize` when needed.

Reply:
I don't care that much about the `textArea` toggle naming (though `textArea` itself is a bit weird), so updated it; `editorProps` is a good one.

Done :+1:

Reply:
I also don't mind about using multi-word/camel-case prop names. But in this case `textArea` was too much about internal implementation rather than compoonent's API.